conans/client/client_cache.py

```diff
@@ -36,6 +37,13 @@ def __init__(self, base_folder, store_folder, output):
         self._default_profile = None
         super(ClientCache, self).__init__(self._store_folder)
 
+    def conanfile_lock(self, conan_ref):
```

Comment:
What folder exactly locks the `conanfile_lock`?

Reply:
The full package root.

Reply:
A comment or rename containing "root" would help me.